iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/BicepLexicalGrammar.java

```diff
@@ -69,7 +72,8 @@ public static LexerlessGrammarBuilder createGrammarBuilder() {
   }
 
   private static void punctuators(LexerlessGrammarBuilder b) {
-    b.rule(Punctuator.EQU).is(Punctuator.EQU.getValue());
+    Stream.of(Punctuator.EQU).forEach(
+      p -> b.rule(p).is(SPACING, p.getValue()).skip());
```

Comment:
Not quite a comment, but rather a question. Does this line mean that spaces surrounding equality signs will be attached to them in the AST? If so, shouldn't it be done for other tokens, e.g. for keywords?

Reply:
No, spaces will be not attached to AST. Usually, we don't need spacing in AST (exception: Docker).
For now, there is no need to do it for keywords - baby steps.

Reply:
So `skip()` basically means that the equality sign along with optional spaces will be a `SyntaxToken` and not a dedicated class of AST node?